builder/main.py

```
@@ -1017,43 +1171,226 @@ def download_littlefs(target, source, env):
+                    if file_size > 0 and cluster_low >= 2:
+                        file_data = read_cluster(cluster_low)[:file_size]
+                    else:
+                        file_data = b''
+                    
+                    output_file = unpack_path / filename
+                    output_file.write_bytes(file_data)
+                    
+                    print(f"  FILE: {filename} ({file_size} bytes)")
+                    extracted_count += 1
+                    
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)